holidays/countries/cambodia.py

```diff
@@ -278,6 +279,16 @@ def _populate_public_holidays(self):
         self._add_holiday(bon_om_touk, _timedelta(bon_om_touk_date, -1))
         self._add_holiday(bon_om_touk, _timedelta(bon_om_touk_date, +1))
 
+        # ទិវា​សន្តិភាព​នៅ​កម្ពុជា​
```

Comment:
```suggestion
        # ទិវា​សន្តិភាព​នៅ​កម្ពុជា​
```
I think zero-width spaces is excessive (here and below in this name).

Reply:
Nice catch 👍